raiden/tests/unit/test_sqlite.py

```diff
@@ -362,6 +363,58 @@ def test_get_event_with_balance_proof():
     storage.close()
 
 
+def test_get_state_change_with_transfer_by_secrethash():
+    serializer = JSONSerializer()
+    storage = SerializedSQLiteStorage(":memory:", serializer)
+
+    secret1, secrethash1 = factories.make_secret_with_hash()
+    channels = factories.mediator_make_channel_pair()
+    transfer1 = factories.create(
+        factories.LockedTransferSignedStateProperties(
+            secret=secret1,
+            target=channels.partner_address(1),
+            initiator=channels.partner_address(0),
+        )
+    )
+    state_change1 = factories.mediator_make_init_action(channels, transfer1)
+
+    secret2, secrethash2 = factories.make_secret_with_hash()
```

Comment:
The difference between state_change1 and state_change2 take a while to spot. Could you add a tine comment or choose names like `state_change_mediator` `state_change_target`?

Reply:
Yes, I changed the names.